motor/platform/platform.proto

```diff
@@ -7,8 +7,6 @@ option go_package = "go.mondoo.com/cnquery/motor/platform";
 
 message Platform {
   string name = 1;
-  // FIXME: remove in v8.0 vv
```

Comment:
removed due to this indicator, the rest is cleanup

Reply:
does this get stored as proto somewhere. Do you need to mark 2 as unusable?